modules/alerting/main.tf

```
@@ -240,6 +240,56 @@ resource "google_monitoring_alert_policy" "service_failure_rate" {
   project = var.project_id
 }
 
+resource "google_monitoring_alert_policy" "service_failure_rate_eventing" {
+  # In the absence of data, incident will auto-close after an hour
+  alert_strategy {
+    auto_close = "3600s"
+  }
+
+  combiner = "OR"
+
+  conditions {
+    condition_prometheus_query_language {
+      duration            = "${var.failure_rate_duration}s"
+      evaluation_interval = "60s"
+
+      // Using custom prometheus metric to avoid counting failed health check 5xx, should be a separate alert
+      // First part of the query calculates the error rate (5xx / all) and the rate should be greater than var.failure_rate_ratio_threshold
+      // Second part ensures services has non-zero traffic over last 5 min.
+      query = <<EOT
+        (sum by (service_name)
+           (rate(http_request_status_total{service_name!~"${join("|", var.failure_rate_exclude_services)}", code=~"5..", ce_type=~"dev.chainguard.*"}[1m]))
+         /
+         sum by (service_name)
+           (rate(http_request_status_total{service_name!~"${join("|", var.failure_rate_exclude_services)}", ce_type=~"dev.chainguard.*"}[1m]))
+        ) > ${var.failure_rate_ratio_threshold}
+        and
+        sum by (service_name)
+          (rate(http_request_status_total{service_name!~"${join("|", var.failure_rate_exclude_services)}", ce_type=~"dev.chainguard.*"}[5m]))
+        > 0.0001
+      EOT
+    }
+
+    display_name = "eventing services 5xx failure rate above ${var.failure_rate_ratio_threshold}"
+  }
+
+  display_name = "eventing services 5xx failure rate above ${var.failure_rate_ratio_threshold}"
+
+  documentation {
+    // variables reference: https://cloud.google.com/monitoring/alerts/doc-variables#doc-vars
+    subject = "Cloud Run service $${metric_or_resource.labels.service_name} had 5xx error rate above ${var.failure_rate_ratio_threshold} for ${var.failure_rate_duration}s"
```

```suggestion
    subject = "Eventing service $${metric_or_resource.labels.service_name} had 5xx error rate above ${var.failure_rate_ratio_threshold} for ${var.failure_rate_duration}s"
```
should we make this distinction? (maybe using more accurate language than I suggested)

---

the service should be included, which should be sufficient.
naming it eventing should reinforce that.